OpenRA.Mods.Common/Projectiles/NukeLaunch.cs

```diff
@@ -90,18 +93,25 @@ public void Tick(World world)
 
 			if (!isLaunched)
 			{
-				anim.PlayRepeating(upSequence);
 				if (weapon.Report != null && weapon.Report.Any())
 					Game.Sound.Play(SoundType.World, weapon.Report, world, pos);
 
-				world.ScreenMap.Add(this, pos, anim.Image);
+				if (!string.IsNullOrEmpty(image))
```

Comment:
The code here doesn't need to know about the details when an animation is created or not, so I suggest:
```suggestion
				if (anim != null)
```